tests/test.desc-desc.js

```diff
@@ -2,6 +2,7 @@ const expect = require("chai").expect;
 const parseString = require('xml2js').parseString;
 const path = require('path');
 const fs = require('fs');
+const helpers = require('./test_helper');
```

Comment:
I think having a whole "helper" module for just a single function in too much. Better whenever needed just explicitly do:
```
testFile.replaceAll(path.sep, '/')
```

Notice, it is very clear and not more complicated then `helpers.normalizePath(testFile)`